### PR TITLE
Zeldazackman henk

### PR DESCRIPTION
**Pull request by Zeldazackman**
## About The Pull Request

Makes clown horns honk once stepped on, also allows for the modification of the horn's honk sound via var editing for badmin use.

## Why It's Good For The Game

Because clown things.

## Changelog
:cl:
add: The ability to excite and infuriate just by running over clown horns.

del: Removed the unfun from clown horns

fix: Fixed the existence of silence ever existing once people figure out how this works.

### CODE REVIEW

**Review by nik707 — APPROVED** [review]
yes

**Comment by buffyuwu** [comment]
need to add your honk_sound to this PR as well if you're adding a new one, or adjust the path

File: code/game/objects/items/weapons/clown_items.dm [hunk 1]
```diff
@@ -69,11 +69,20 @@
...
 		src.add_fingerprint(user)
 		spawn(20)
 			spam_flag = 0
 	return
+
+/obj/item/weapon/bikehorn/Crossed(atom/movable/AM as mob|obj)
+	if(AM.is_incorporeal())
+		return
+	if(istype(AM, /mob/living))
+		playsound(src, honk_sound, 50, 1)
```
Comment (buffyuwu) [inline code/game/objects/items/weapons/clown_items.dm:88]:
```suggestion
		playsound(src, 'honk_sound', 50, 1)
```

**Comment by Zeldazackman** [comment]
I'm too smoothbrained right now to figure out why it compiled locally but keeps failing here

**Pull request closed without merging**